### PR TITLE
RELATED: RAIL-2784 fix InsightView docs and examples

### PR DESCRIPTION
**Pull request by no23reason**
Also, made sure Supported Versions are available in the "next" section as well.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/546/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/323/ : SUCCESS in 4s

File: docs/10_vis__insight_view.md [hunk 1]
```diff
@@ -75,7 +76,7 @@ clearInsightViewCaches();
 
 | Name | Required? | Type | Description |
 | :--- | :--- | :--- | :--- |
-| identifier | false | string | The identifier of the visualization to be rendered. Can be omitted if the visualization URI is present. |
+| insight | true | ObjRef or string | The reference to the visualization to be rendered or its identifier. |
```
Comment (evgeniabogranskaya) [inline docs/10_vis__insight_view.md:79]: `The reference to or the identifier of the visualization the to be rendered`

no full stop please =-)

File: website/versioned_docs/version-8.0.0/10_vis__insight_view.md [hunk 2]
```diff
@@ -76,7 +77,7 @@ clearInsightViewCaches();
 
 | Name | Required? | Type | Description |
 | :--- | :--- | :--- | :--- |
-| identifier | false | string | The identifier of the visualization to be rendered. Can be omitted if the visualization URI is present. |
+| insight | true | ObjRef or string | The reference to the visualization to be rendered or its identifier. |
```
Comment (evgeniabogranskaya) [inline website/versioned_docs/version-8.0.0/10_vis__insight_view.md:80]: `The reference to or the identifier of the visualization the to be rendered`

no full stop please =-)

**Comment by evgeniabogranskaya** [comment]
Suggested a minor edit
LGTM

File: docs/10_vis__insight_view.md (hunk 1 above)
Comment (no23reason) [reply docs/10_vis__insight_view.md]: Fixed, thanks 🙂

File: website/versioned_docs/version-8.0.0/10_vis__insight_view.md (hunk 2 above)
Comment (no23reason) [reply website/versioned_docs/version-8.0.0/10_vis__insight_view.md]: Fixed, thanks 🙂

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/547/ : SUCCESS in 1m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/324/ : SUCCESS in 5s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/548/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/66/ : SUCCESS in 21s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/159/ : SUCCESS in 23s

**Pull request merged**